packages/web/src/scss/components/Pagination/README.md

```diff
@@ -0,0 +1,67 @@
+# Pagination
+
+First item selected:
+
+```html
+<div class="Pagination">
```

Comment:
This might be `<nav>` tag with proper aria?

Reply:
Also, it probably should be `<ul>` list of links. What do you think? 

Google `pagination a11y` gets me these results:
https://www.a11ymatters.com/pattern/pagination/
https://design-system.w3.org/components/pagination.html
https://a11y-style-guide.com/style-guide/section-navigation.html
https://developer.visa.com/pages/accessibility/navigation/nav-007

Reply:
There was completely missing the "accessibility" code part since there has never been need for this.
It's been added, please check now.